libs/examples/src/chart/chart-behavior-switch-example/chart-behavior-switch-example.ts

```diff
@@ -15,112 +15,110 @@
  */
 
 import { Component } from '@angular/core';
-
 import { generateData } from '../chart-data-utils';
 
+const cpuUsageSeries: Highcharts.IndividualSeriesOptions[] = [
```

Comment:
I think we should move these array back into the class as a prop so it appears in the example source in barista.

Reply:
Yep, also thought about that... was not sure if this is critical information for the example. But I'll move it back again! :)